changelog/0032-upgrade-elasticsearch.yml

```diff
@@ -0,0 +1,50 @@
+---
+title: ES upgrade from 1.7.6 to 2.4.6
+key: upgrade-elasticsearch
+date: 2020-02-05
+optional_per_env: yes
+update_steps: |
+  1.stop the site
+  commcare-cloud <env> downtime start
```

Comment:
@sanjay2916 can you put command and config changes inside code blocks

Reply:
sure